NEWS.md

```diff
@@ -2,6 +2,8 @@
 
 * Now supports interface users 15.0, 16.0 (MODINV-725)
 * Extend instance contributors schema with Authority ID ([MODINV-729](https://issues.folio.org/browse/MODINV-729))
+* Delete instance/item by CQL ([MODINV-731](https://issues.folio.org/browse/MODINV-731))
+* Provides inventory 12.0, requires instance-storage 9.0, item-storage 10.0, holdings-storage 2.0 ... 6.0.
```

Comment:
Please split these entries onto separate lines in the news, as that is the existing convention and easier to read.